netbout-web/src/main/java/com/netbout/rest/bout/TkIndex.java

```diff
@@ -83,75 +84,81 @@ public Response act(final Request req) throws IOException {
         final Href home = new Href("/b").path(bout.number());
         final RqWithDefaultHeader request =
             new RqWithDefaultHeader(req, HttpHeaders.ACCEPT, "text/xml");
-        return new RsPage(
-            "/xsl/bout.xsl",
-            this.base,
-            request,
-            new XeAppend(
-                "bout",
-                new XeDirectives(
-                    new Directives()
-                        .add("number")
-                        .set(Long.toString(bout.number()))
-                        .up()
-                        .add("title").set(bout.title()).up()
-                        .add("unread")
-                        .set(Long.toString(bout.messages().unread()))
-                        .up()
-                        .add("subscription")
-                        .set(String.valueOf(bout.subscription()))
-                ),
+        try {
```

Comment:
@2686747 we shouldn't be needing all these changes anymore, can't we just revert everything changed in this file ?